lib/plugins/inventory.js

```diff
@@ -212,7 +213,8 @@ function inject (bot, { version, hideErrors }) {
         (nbt != null && window.selectedItem.nbt !== nbt)) {
         // we are not holding the item we need. click it.
         const sourceItem = window.findItemRange(sourceStart, sourceEnd, itemType, metadata, false, nbt)
-        if (!sourceItem) throw new Error(`missing source item ${itemType}:${metadata} in (${sourceStart},${sourceEnd})`)
+        assert(mcData.itemsArray.find(x => x.id === itemType), 'Invalid itemType')
```

Comment:
Put that in a variable to avoid duplicating

Reply:
fixed